### ISSUE TITLE
Accessibility, Section 508 Compliance: HTML Tables missing ARIA attributes

### ISSUE THREAD

**Issue by jdm4344** [Azure/api-management-developer-portal#1379]
## Bug description

In the Operation Details Widget, response tables have no [role=columnheader] or [role=rowheader] attributes for cells. Definition tables are also missing the [role=row] attribute.

## Reproduction steps

N/A

## Expected behavior

HTML tables and child elements have the ARIA attributes necessary to be properly recognized by accessibility software.

## Is your portal managed or self-hosted?

Managed

## Environment

 - Operating system: MacOS 10.15.7 - Macbook Pro 15-inch, 2017 model
 - Browser: Google Chrome
 - Version: 91

## Additional context

Related Section 508 test step: https://section508coordinators.github.io/TrustedTester/tables.html#check-131-cell-header-association

Missing [role=columnheader] / [role=rowheader]
<img width="1680" alt="Tables - Header" src="https://user-images.githubusercontent.com/31706474/125804958-4bc337ec-344b-4941-b412-c39fad623d33.png">

Missing[role=row]
<img width="1671" alt="Tables - Role" src="https://user-images.githubusercontent.com/31706474/125804267-b1e254fe-82af-4e5b-9c45-ae4b89c71878.png">

**Comment by christopherwoodland** [comment]
Thank you for fixing this issue!
